README.md

```diff
@@ -184,30 +184,216 @@ Depending on the distribution you use, it might be possible to install lldap
 community contributed.
 
-#### Debian, CentOS Fedora, OpenSUSE, Ubuntu
-
-The package for these distributions can be found at [LLDAP OBS](https://software.opensuse.org//download.html?project=home%3AMasgalor%3ALLDAP&package=lldap).
-- When using the distributed package, the default login is `admin/password`. You can change that from the web UI after starting the service.
-
-#### Arch Linux
-
-Arch Linux offers unofficial support through the [Arch User Repository
-(AUR)](https://wiki.archlinux.org/title/Arch_User_Repository).
-Available package descriptions in AUR are:
-
-- [lldap](https://aur.archlinux.org/packages/lldap) -  Builds the latest stable version.
-- [lldap-bin](https://aur.archlinux.org/packages/lldap-bin) - Uses the latest
-  pre-compiled binaries from the [releases in this repository](https://github.com/lldap/lldap/releases).
-  This package is recommended if you want to run lldap on a system with
-  limited resources.
-- [lldap-git](https://aur.archlinux.org/packages/lldap-git) - Builds the
-  latest main branch code.
-
-The package descriptions can be used
-[to create and install packages](https://wiki.archlinux.org/title/Arch_User_Repository#Getting_started).
-Each package places lldap's configuration file at `/etc/lldap.toml` and offers
-[systemd service](https://wiki.archlinux.org/title/systemd#Using_units)
-`lldap.service` to (auto-)start and stop lldap.
+Each package offers a [systemd service](https://wiki.archlinux.org/title/systemd#Using_units) `lldap.service` to (auto-)start and stop lldap.
+
+<details>
+<summary><b>Arch</b></summary>
+<br>
+  Arch Linux offers unofficial support through the <a href="https://wiki.archlinux.org/title/Arch_User_Repository">Arch User Repository (AUR)</a>.<br>
+  The package descriptions can be used <a href="https://wiki.archlinux.org/title/Arch_User_Repository#Getting_started">to create and install packages</a>.<br><br>
+  Maintainer: <a href="https://github.com/Zepmann">@Zepmann</a><br>
+  Support: <a href="https://github.com/lldap/lldap/discussions">Discussions</a><br>
+  Package repository: <a href="https://aur.archlinux.org/packages">Arch user repository</a><br>
+<table>
+  <tr>
+    <td>Available packages:</td>
+    <td><a href="https://aur.archlinux.org/packages/lldap">lldap</a></td>
+    <td>Builds the latest stable version.</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="https://aur.archlinux.org/packages/lldap-bin">lldap-bin</a></td>
+    <td>Uses the latest pre-compiled binaries from the <a href="https://aur.archlinux.org/packages/lldap-bin">releases in this repository</a>.<br>
+        This package is recommended if you want to run lldap on a system with limited resources.</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td><a href="https://aur.archlinux.org/packages/lldap-git">lldap-git</a></td>
+    <td>Builds the latest main branch code.</td>
+  </tr>
+</table>
+LLDPA configuration file: /etc/lldap.toml<br>
+</details>
+<details>
+<summary><b>Debian</b></summary>
```

Comment:
Should we collapse Debian/Ubuntu/opensuse and so on? It's just the same thing repeated

Reply:
Right now these sections contain redundant information, that is right.
I mean, that is because I am maintaining all of them. But this might change in future.
We may get lldap accepted in official repositories, someone else might take over for some distributions or I might stop maintaining some of them.

I also think it is more welcoming for people who are looking for help with their specific distro.
Like, I am a Debain guy, I prefer a button that says, hit me if you use Debian over a section that applies to several distros including mine.